samples/benchmark/gamelife/kotlin/src/main/kotlin/godot/samples/benchmark/gamelife/game.kt

```diff
@@ -0,0 +1,176 @@
+package godot.samples.benchmark.gamelife
```

Comment:
When you moved the other classes in this script, you should name the file the same as the Class it contains